docs/cli/getting-started.md

```diff
@@ -23,6 +23,7 @@ With [v0.14.0](https://github.com/vmware-tanzu/tanzu-framework/releases/tag/v0.1
 The API driven plugin discovery feature is enabled as default method to install the plugins. So, most users will just need to install the Tanzu CLI binary. The plugins can be installed and configured with `tanzu plugin sync` command. Learn more about this feature in the [design docs](../design/context-aware-plugin-discovery-design.md).
 
 ##### macOS/Linux
+###### Option 1: Manual download CLI binary from Github Releases
```

Comment:
nit: Please use proper casing for GitHub.

Reply:
Changed the casing of GitHub as suggested.